src/library/SubmitTx/ManualSign.tsx

```diff
@@ -139,6 +141,14 @@ export const ManualSign = ({
                   ? fallbackMessage
                   : statusCodeTitle)
               : fallbackMessage}
+            {defaultMessage ===
```

Comment:
There should be `helpKey` returned by determineStatusCode, and that helpKey should be tested here.